static/app/views/organizationIntegrations/integrationAlertContainer.tsx

```diff
@@ -0,0 +1,7 @@
+import styled from '@emotion/styled';
```

Comment:
@scefali is this the right place for this styled component? it looks like we only use it in one place. can we just define it there?

Reply:
@NisanthanNanthakumar It's used in getsentry